neo/Persistence/Store.cs

```diff
@@ -36,6 +36,7 @@ public abstract class Store : IPersistence
         public abstract MetaDataCache<HashIndexState> GetBlockHashIndex();
         public abstract MetaDataCache<HashIndexState> GetHeaderHashIndex();
         public abstract void Put(byte prefix, byte[] key, byte[] value);
```

Comment:
Nothing actually uses this method anymore does it?

Reply:
We only need the `PutSync` method now then, so we could remove the `Put` method altogether.

Reply:
We can keep it for future use.